packages/stencil/components/file/src/components/osds-file/osds-file-header/osds-file-header.tsx

```diff
@@ -0,0 +1,117 @@
+import { Component, Event, EventEmitter, h, Prop } from '@stencil/core';
+import { odsFileDefaultAttributes, OdsIconName, OdsIconSize,
+  OdsFileHeader, OdsFileHeaderController } from '@ovhcloud/ods-core';
+import { OdsThemeColorIntent } from '@ovhcloud/ods-theming';
+
+@Component({
+  tag: 'osds-file-header',
+  styleUrl: '../osds-file.scss',
+  shadow: true
+})
+export class OsdsFileHeader implements OdsFileHeader {
+  controller: OdsFileHeaderController = new OdsFileHeaderController(this);
+
+  @Prop({ reflect: true }) public isSuccessful = false;
+  @Prop({ reflect: true }) public disabled = odsFileDefaultAttributes.disabled;
+  @Prop({ reflect: true }) public errorMessage = odsFileDefaultAttributes.errorMessage;
+  @Prop({ reflect: true }) public acceptedTypes = odsFileDefaultAttributes.acceptedTypes;
+
+  @Event() filesSelected!: EventEmitter<File[]>;
+
+  componentWillRender(): void {
+    this.controller.validateAttributes();
+  }
+
+  emitFiles = (files: File[]) => {
+    this.filesSelected.emit(files);
+  }
+
+  acceptedExtensions = () => {
+    return this.controller.getAcceptedTypes()
+  }
+
+  handleFilesSelected = (e: Event) => {
+    this.controller.getFilesFromSelection(e)
+  }
+  render() {
+      if(this.errorMessage) {
+        return <div class='ods-file__dropzone--errored__header'>
+          <div class='ods-file__dropzone--errored__header__message'>
+            <div class='ods-file__dropzone--errored__header__icon-wrapper'>
+              <osds-icon name={OdsIconName.CLOSE}
+                         color={OdsThemeColorIntent.error}
+                         size={OdsIconSize.xs}
+              />
+            </div>
+            <span class='ods-file__dropzone__header__title__label'>{this.errorMessage}</span>
+          </div>
+          <section>
+            <div class='ods-file__dropzone__header__title'>
+              <span class='ods-file__dropzone__header__title__label '>Glisser-déposer une pièce jointe</span>
```

Comment:
We are not managing any labels for now as we don't have any i18n system.
Static strings should be set as attributes